modules/virt-importing-rhel-image-boot-source-web.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-importing-rhel-image-boot-source-web_{context}"]
+= Importing a Red Hat Enterprise Linux image as a boot source
+
+You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by specifying the URL address for the image.
+
+.Prerequisites
+* To import a RHEL image via a URL, access to the web server with the operating system image is required. For example: the Red Hat Enterprise Linux web page with images.
```

Comment:
Suggestion: "You must have access to the web server with the operating system image. For example: the Red Hat Enterprise Linux web page with images."

Reply:
Updated the content.